src/components/form/HdRadioIndicator.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div
+    class="indicator"
+    :class="{
+      ['indicator--selected']: checked,
+      ['indicator--disabled']: disabled,
+      ['indicator--invalid']: invalid,
+    }"
+  ></div>
+</template>
+
+<script>
+export default {
+  name: 'HdRadioIndicator',
+  props: {
+    checked: {
+      type: Boolean,
+      required: true,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    invalid: {
+      type: Boolean,
+      default: false,
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import "homeday-blocks/src/styles/mixins.scss";
+
+.indicator {
```

Comment:
Same here, I propose that we remove the `scoped` keyword. This could create an issue because `indicator` is kind of generic, but maybe we should make the class more specific? Perhaps `radio-indicator`?

Reply:
Thanks for your suggestion man =D here are the changes

9bb0eee